be/src/exec/pipeline/pipeline.cpp

```diff
@@ -102,7 +103,8 @@ void Pipeline::instantiate_drivers(RuntimeState* state) {
         if (auto* scan_operator = driver->source_scan_operator()) {
             scan_operator->set_workgroup(workgroup);
             scan_operator->set_query_ctx(query_ctx->get_shared_ptr());
-            if (dynamic_cast<ConnectorScanOperator*>(scan_operator) != nullptr) {
+            if (dynamic_cast<ConnectorScanOperator*>(scan_operator) != nullptr ||
+                dynamic_cast<SchemaScanOperator*>(scan_operator) != nullptr) {
```

Comment:
prefer port to scan_executor? some user online disable the connector scan executor?

Reply:
are you sure??? how to disable it